### DISCUSSION TITLE
Arithmetic Sanity: Safeguarding Transactions with Arithmetic Checks #CryptoGames #NFT #MintTokens

### DISCUSSION THREAD

**Discussion by richafltr** [stellar/sorobounty-spectacular#20] [General]
In an increasingly digital world, the security of transactions, especially in the decentralized space, becomes paramount. One innovative solution is the integration of basic arithmetic checks prior to transaction submission, ensuring a second layer of user presence verification. Named "Arithmetic Sanity", this concept combines user interaction with basic cognitive tasks to create a safer transactional environment.

Let's explore the importance and execution of this concept.

GitHub Code for Dapp and Readme: https://github.com/richafltr/arithmetic-sanity-dapp.git
Live Game: 
Medium Tutorial for the Dapp:
Yotube Video Content:

**Comment by briwylde08** [comment]
Thank you for your Sorobounty Spectacular submission! We will review and get back to you within a week.

**Comment by briwylde08** [comment]
Hi there! Thank you for your submission. Unfortunately, this doesn’t quite follow the eligibility guidelines for the Sorobounty Spectacular program.

The minimum requirement for a submission is original dapp code that is publicly available in a GitHub repo with published educational content (such as written tutorials and video walkthroughs) in a GitHub README, [dev.to](http://dev.to/) or Medium article, or YouTube video.

Please let me know if you have any questions!
